R/run_eggla_gwas.R

```diff
@@ -586,61 +619,48 @@ run_eggla_gwas <- function(
           idcol = "trait_model"
         ),
         old = function(x) sub("^#", "", x)
-      )[TEST %in% "ADD" & !is.na(P), -c("TEST")]
+      )[
+        TEST %in% "ADD" & !is.na(P),
+        -c("TEST", "T_STAT", "OBS_CT")
```

Comment:
Remove default PLINK2 columns, especially `OBS_CT` which is not the number of analysed samples.